js/safari-user-agent-detect.modernizr.js

```diff
@@ -12,5 +12,10 @@
 
     return ua.indexOf('Safari/') >= 0 && ua.indexOf('Chrome/') == -1;
   });
+
+  Modernizr.addTest('safari-idevice', function () {
+      var ua = window.navigator.userAgent;
+      return ua.indexOf('Safari/') >= 0 && ua.indexOf('iP') >= 0;
```

Comment:
Is this distinct enough? I think that it'd match Safari-resembling UAs on iThings (eg. Chrome on iOS), and any

Reply:
That might be what we want since everything on iOS is forced to use the same WebKit engine, presuambly with the same SVG capabilities.

Reply:
Good question, this came up [before](https://repo.elab.warwick.ac.uk/projects/START/repos/app/pull-requests/1085/overview?commentId=17959)

Reply:
> That might be what we want since everything on iOS is forced to use the same WebKit engine, presuambly with the same SVG capabilities.

Makes sense, in which case, just rename the test (for future sanity).